pkg/bpf/detect_windows.go

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package bpf
+
+import "fmt"
+
+func HasOverrideHelper() bool {
+	return false
+}
+
+func HasSignalHelper() bool {
+	return false
+}
+
+func HasBuildId() bool {
+	return false
+}
+
+func HasModifyReturn() bool {
+	return false
+}
+
+func HasModifyReturnSyscall() bool {
+	return false
+}
+
+func HasProgramLargeSize() bool {
+	return false
+}
+
+func HasLinkPin() bool {
+	return false
+}
+
+func HasKprobeMulti() bool {
+	return false
+}
+
```

Comment:
there's some more Has* functions in current detect_linux.go 
```
HasLSMPrograms
HasMissedStatsKprobeMulti
HasMissedStatsPerfEvent
```

Reply:
Yes, they were removed as they do not make sense on Windows and were not called from common code. Removing them did not break compilation.